services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedTenantService.java

```diff
@@ -0,0 +1,528 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.deviceregistry.mongodb.service;
+
+import static io.vertx.core.impl.CompositeFutureImpl.all;
```

Comment:
Check style error due to static import. Static imports are not allowed except the tests.

`[ERROR] src/main/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedTenantService.java:[16] (imports) AvoidStaticImport: Using a static member import should be avoided - io.vertx.core.impl.CompositeFutureImpl.all.`

Reply:
fixed